python/lancedb/table.py

```diff
@@ -300,6 +300,19 @@ def create(cls, db, name, data, schema=None, mode="create"):
         lance.write_dataset(data, tbl._dataset_uri, mode=mode)
         return tbl
 
+    @classmethod
+    def open(cls, db, name):
+        tbl = LanceTable(db, name)
```

Comment:
instead we could do 
```python
cls(db, name)
```

This way child classes can use this method too without having to reimplement